### PR TITLE
Bugfix for new Liquid tag

### PR DESCRIPTION
**Pull request by shopmike**
These are needed to pass tests for the Liquid tag that was merged

1. Resolve echo tag issues

@pushrax @tobi @fw42 @ashmaroli

### CODE REVIEW

File: test/integration/tags/echo_test.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class EchoTest < Minitest::Test
   include Liquid
 
   def test_echo_outputs_its_input
-    assert_template_result('BAR', <<~LIQUID, { 'variable-name' => 'bar' })
-      {%- echo variable-name | upcase -%}
-    LIQUID
+    assert_template_result('BAR', <<-LIQUID, { 'variable-name' => 'bar' })
+{%- echo variable-name | upcase -%}
```
Comment (fw42) [inline test/integration/tags/echo_test.rb]: this indentation kinda sucks 😢

Comment (shopmike) [reply test/integration/tags/echo_test.rb]: The alternative to this approach is bumping Ruby to 2.3

**Comment by fw42** [comment]
The `render` vs. `render_to_output_buffer` change makes sense to me, but I don't get why the heredoc stuff needs to change (for the worse imho). Can you explain?

File: test/integration/tags/echo_test.rb (hunk 1 above)
Comment (fw42) [reply test/integration/tags/echo_test.rb]: What does that have to do with those new tags though?

Comment (shopmike) [reply test/integration/tags/echo_test.rb]: `<<~HEREDOC` is only support in 2.3+

Comment (fw42) [reply test/integration/tags/echo_test.rb]: I'd rather not use heredoc at all here if it forces us to use this ugly indentation style

Comment (shopmike) [reply test/integration/tags/echo_test.rb]: Currently liquid is `2.1+`

Comment (shopmike) [reply test/integration/tags/echo_test.rb]: I can use https://github.com/JuanitoFatas/String-strip_heredoc-patch/blob/master/strip_heredoc.rb on this to make it pretty

**Comment by shopmike** [comment]
I might just pull the HEREDOC stuff out and we can just ship https://github.com/Shopify/liquid/pull/1131 which will resolve that part

**Comment by ashmaroli** [comment]
Since #1131 will get merged in the near future, the heredoc changes can be undone. The tests may fail, but the diff will be much cleaner and focused.

**Comment by shopmike** [comment]
I think everyone sort of agrees we should just merge https://github.com/Shopify/liquid/pull/1131 and it will resolve the use of the heredoc here.

**Comment by shopmike** [comment]
This is now ready to go when someone is ready.

**Pull request merged**